package.json

```diff
@@ -13,7 +13,7 @@
     "test": "mocha --use_strict --recursive",
     "tdd": "mocha --use_strict --recursive -w",
     "watch": "mocha --use_strict --recursive -w",
-    "coverage": "nyc mocha --use_strict --recursive"
+    "coverage": "nyc mocha --use_strict --recursive && nyc report --reporter=text-lcov | coveralls"
```

Comment:
This script won't be possible to run on Windows. Not a huge thing in case we don't want to fix it.

If we want to make it cross-platform we could probably create a standalone script which either uses the programmatic API of `nyc`, `mocha` and `coveralls` or makes an OS check and calls an appropriate shell or bat script.

Reply:
It did run for me (Powershell) but I will try cmd when I get home :) Might fix it before merge or save that improvement for a later PR. Thanks for the feedback though didn't even think off cross os support!

Reply:
Works on Windows for both Powershell and Cmd, I think npm/node solves both `&&` and `|` 🍾